### PR TITLE
ci: add a retry for terraform apply

### PR DESCRIPTION
**Pull request by alexandre-allard**
**Component**: ci

**Context**: see #2401 

**Summary**:
Let see if it decreases a bit flakiness

**Acceptance criteria**: 

---

Closes: #2401

### CODE REVIEW

**Review by Ebaneck — DISMISSED** [review]

**Comment by gdemonet** [comment]
@alexandre-allard-scality This doesn't seem to have any effect - found a build failure (sent you the link) where we get the "existing key pair" conflict on all three retries.

**Comment by alexandre-allard-scality** [comment]
No longer needed as the real issue has been fixed by #2439

File: eve/main.yml [hunk 1]
```diff
@@ -385,7 +385,10 @@ models:
       haltOnFailure: true
   - ShellCommand: &terraform_apply
       name: Spawn openstack virtual infra
-      command: terraform apply -auto-approve
+      command: |-
+        for _ in $(seq 1 3); do
+            terraform apply -auto-approve && break
+        done
```
Comment (gdemonet) [inline eve/main.yml]: Let's add a few minutes of sleep time between attempts

**Review by gdemonet — CHANGES_REQUESTED** [review]

File: eve/main.yml (hunk 1 above)
Comment (alexandre-allard) [reply eve/main.yml]: Yes, why not.

**Review by gdemonet — APPROVED** [review]

**Pull request merged**